src/schema/v2/homeView/sections.ts

```diff
@@ -53,9 +63,10 @@ export const TrendingArtists: HomeViewSection = {
 }
 
 const sections: HomeViewSection[] = [
-  RecentlyViewedArtworks,
   AuctionLotsForYou,
   NewWorksForYou,
+  NewWorksFromGalleriesYouFollow,
+  RecentlyViewedArtworks,
```

Comment:
👀

Reply:
I just ordered this alphabetically  - recently viewed was the first rail